pkgs/applications/graphics/drawing/default.nix

```diff
@@ -0,0 +1,72 @@
+{ stdenv
+, fetchFromGitHub
+, desktop-file-utils
+, file
+, gobject-introspection
+, pkgconfig
+, appstream-glib
+, meson
+, ninja
+, python3
+, gtk3
+, wrapGAppsHook
+, gettext
+, glib
+, python3Packages
+}:
+
+stdenv.mkDerivation rec {
+  pname = "drawing";
+  version = "0.4.10";
+
+  src = fetchFromGitHub {
+    owner = "maoschanz";
+    repo = "${pname}";
+    rev = "${version}";
+    sha256 = "1i8b7hkv9484c0pr2lag6xkf4mq5b2gwcawppgfkf1l8971zxqik";
+  };
+
+  nativeBuildInputs = [
+    wrapGAppsHook
+    gobject-introspection
+    pkgconfig
+    appstream-glib
+    meson
+    ninja
+    python3
+    gtk3
+    gettext
+    glib
+    python3Packages.wrapPython
+  ];
+
+  buildInputs = [
```

Comment:
you're missing
* gdk-pixbuf https://github.com/maoschanz/drawing/blob/eb2397bccac38c8d671d33a66a8bcc8de5352927/src/tools/utilities_overlay.py#L19
* pango https://github.com/maoschanz/drawing/blob/eb14358b7ff60680ec72d7f172724d1fe88264f2/src/image.py#L19

It's also not good that the meson build system doesn't do `dependency` for anything, or check for the required python3 modules.

Reply:
absolutely right : done

Reply:
It seems you removed `glib` from here, which it for sure needs, but also added `gdk-pixbuf` and `pango` to nativeBuildInputs as well. This comment was just for buildInputs, because that's where the runtime dependencies go.

Reply:
exact, seems correct now